connector.yml

```diff
@@ -0,0 +1,24 @@
+---
+version: '3'
+
+services:
+  transmart-variant-store-connector:
+    container_name: transmart-variant-store-connector
+    image: thehyve/transmart-variant-store-connector
+    ports:
+    - 9060:9060
+    environment:
+      KEYCLOAK_SERVER_URL: ${KEYCLOAK_SERVER_URL:?Please specify a Keycloak url.}
+      KEYCLOAK_REALM: ${KEYCLOAK_REALM:?Please specify a Keycloak realm.}
+      KEYCLOAK_CLIENT_ID: ${KEYCLOAK_CLIENT_ID:?Please specify a Keycloak resource.}
+      TRANSMART_API_SERVER_URL: ${TRANSMART_API_SERVER_URL:?Please specify a Transmart url.}
```

Comment:
This should be the internal address `http://transmart-api-server:8081` (see, e.g., `gb-backend`)